guide/case-studies/payjoin.md

```diff
@@ -0,0 +1,317 @@
+
+We analyzed the payjoin user flow in BlueWallet. Read the detailed investigation [here](https://docs.google.com/document/d/1_de2pkMREGpZQwOefQdH-MZNSUPazObgRK1cruiWBmo/edit#bookmark=id.jamlvi37p922).
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/payjoin case study/bluewallet-sender-flow.png"
+   retina = "assets/images/guide/case-studies/payjoin case study/bluewallet-sender-flow@2x.png"
+   mobile = "assets/images/guide/case-studies/payjoin case study/bluewallet-sender-flow-mobile.png"
+   mobileRetina = "assets/images/guide/case-studies/payjoin case study/bluewallet-sender-flow-mobile@2x.png"
+   alt-text = "bluewallet sender flow"
+   width = 800
+   height = 54
+%}
+
+Here are the findings:
+
+- The fee rate drops once transaction size increases during the payjoin process,  which may change estimated confirmation time
+- The flow results in a transaction that has a round amount fee-rate after removing one input, which may be used by chain analysis firms to spot payjoins
+
+#### The Payjoin Sender Flow
+
+We designed a sender flow that attempts to address the issues identified above.
+
+For the purpose of creating the sender flow, we will assume that the receiver only contributes UTXOs but does not contribute to the fees since this allows a simple, automated process for both parties.
+
+### In short, the sender flow outlined here asks the user to choose a fee-range instead of a fee-amount (or fee-rate) while keeping the rest of the user flow almost exactly the same. We use it to set 3 optional parameters specified in BIP-78, which can be used to construct a simple but effective payjoin implementation.
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/payjoin case study/payjoin sender flow.png"
+   retina = "assets/images/guide/case-studies/payjoin case study/payjoin sender flow@2x.png"
+   alt-text = "payjoin sender flow"
+   width = 800
+   height = 374
+%}
+
+The detailed thought process can be read [here](https://docs.google.com/document/d/1IPCQsdoVBGEceUnWhOeEhi-xcNF049RG1tRAchz-Iuc/edit?usp=sharing).
+
+View this flow in [Figjam here](https://www.figma.com/file/69uUDWVc8N9t5Bej8pZEsF/PayJoin-User-Flows?node-id=0%3A1&t=8eOBQa7JofjZ3p3z-1). The detailed process and considerations can be reviewed in this [FigJam file](https://www.figma.com/file/NzMvwyzP7x5jfGmwNUKRov/PayJoin-Process-Flows?node-id=336%3A619&t=BhGymSWCzl2Xm2oO-1).
+
+
+##### The Payjoin handshake
+
+<p class="h3">The term ‘Payjoin handshake’ is being introduced to succinctly describe the process of sending a original PSBT to receiver’s endpoint, receiver contributing their inputs, modifying the transaction as needed and signing the new draft of the transaction.</p>
+
+The above steps are not important to the users involved in the transaction, and do not need specific user action from either the sender or receiver. More generally, the transacting parties need not even be aware that this is occurring.
```

Comment:
I  love to see innovation in the way we can explain the interaction! Way easier to understand what's going on if we can  put consise words to it. However, I think "payjoin *rendezvous*" is better suited than "payjoin handshake" in this context.

In the context of an interactive protocol (like TLS), a handshake refers to a series of steps or messages exchanged between two parties to establish a connection and initialize communication. In payjoin, the communication is already over after the steps described here as a handshake are complete

A rendezvous refers to a mechanism or procedure for two parties to synchronize and agree on the parameters of an interaction.

Reply:
I can see the confusion for a technical reader but the 'handshake' is intended for a colloquial, non-technical purpose/audience. It's also clearer & simpler (to me) than the 'rendezvous'.

Reply:
I'm not even sold on rendezvous. It would be good to say *why* the term needs to be introduced. Did you run into confusion in existing UI where there was no explanation of what was going on? And introducing new vocabulary creates a better experience?

Reply:
I think so, addressed this in the case study text.